test/integration-tests/integration.test.ts

```diff
@@ -27,7 +27,7 @@ const testCases = [
     testDomain: `auto-domain-${RANDOM_STRING}.${TEST_DOMAIN}`,
     testEndpoint: "EDGE",
     testFolder: "auto-domain",
-    testStage: "dev",
+    testStage: "test",
```

Comment:
Is this change intentional?

Reply:
Yep
Just put the same as in the [confix](https://github.com/amplify-education/serverless-domain-manager/blob/5a11e972f5a786b8ee19f684c2545e6a8c61994d/test/integration-tests/auto-domain/serverless.yml#L7)